.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.1.11
+current_version = 1.1.12
```

Comment:
hmm, did you use the tool `bump2version`? 🤔 
Because there are many more files, which need to be bumped

Reply:
this one: https://github.com/c4urself/bump2version